packages/@aws-cdk/aws-rds/test/test.serverless-cluster.ts

```diff
@@ -617,6 +617,41 @@ export = {
 
     test.done();
   },
+
+  'check that clusterArn property works'(test: Test) {
+    // GIVEN
+    const stack = testStack();
+    const vpc = ec2.Vpc.fromLookup(stack, 'VPC', { isDefault: true });
+    const cluster = new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+      vpc,
+    });
+    const exportName = 'DbCluterArn';
+
+    // WHEN
+    new cdk.CfnOutput(stack, exportName, {
+      exportName,
+      value: cluster.clusterArn,
+    });
+
+    // THEN
+    expect(stack).to(haveOutput({
+      exportName,
+      outputValue: {
+        'Fn::Join': [
+          '',
+          [
+            'arn:',
+            { Ref: 'AWS::Partition' },
+            ':rds:us-test-1:12345:cluster:',
+            { Ref: 'DatabaseB269D8BB' },
+          ],
+        ],
+      },
+    }));
```

Comment:
why not directly compare the `cluster.clusterArn` property to the expected value?
Is an output required?

Reply:
No, it's actually not. I just took inspiration from the DocDB constructs tests:

https://github.com/aws/aws-cdk/blob/0297f31ef4224b374a340ba09eeafe963d62e789/packages/%40aws-cdk/aws-docdb/lib/instance.ts#L101

Reply:
I tired:

```ts
test.deepEqual(cluster.clusterArn, 'arn:rds:us-test-1:12345:cluster:DatabaseB269D8BB');
```

but that one fails with:

```
AssertionError: 'arn:${Token[AWS.Partition.3]}:rds:us-test-1:12345:cluster:${Token[TOKEN.13366]}' deepEqual 'arn:rds:us-test-1:12345:cluster:DatabaseB269D8BB'
```

Can you point me to an example on how to compare values directly?

Reply:
I suspect including token placeholders isn't the way to go ;)

Reply:
you'd need to do a `stack.resolve(cluster.clusterArn)` to resolve the tokens 🙂

Reply:
Perfect, thanks :)